citi-onboarding/Agreste-Consultoria-Jr
# feat(feature/keystone-models-2): Include new models

## What I did

- Models of the backend (Depoiments, Services and Company Information);
- Routes;

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout feature/keystone-models-2
   1. git pull origin feature/keystone-models-2
   1. yarn install
   1. code .
   1. yarn start

1. In your browser go to http://localhost:3001/admin

1. Check with the models appear in the keystone interface 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 1h30
